timeseries/setup.py

```diff
@@ -55,7 +57,9 @@
     ],
 }
 
-extras_require["all"] = []
+extras_require["all"] = [
+    "optimum[openvino,onnxruntime,nncf]>=1.17,<1.18",  # for faster CPU inference in pretrained models
```

Comment:
Do we need [neuron suppport](https://huggingface.co/docs/optimum-neuron/index) to speed up inference on Inf2 architecture?

Reply:
Not at the moment, though this will come in v1.2.